### PR TITLE
If No Channels Are Available 

### PR DESCRIPTION
**Pull request by christianmemije**
## Summary

Handles if there are no channels available. 

## TODO

- [ ] Tests

## Screenshot

image

### CODE REVIEW

**Comment by indirectlylit** [comment]
For the copy, maybe:

----

## No Content Available

[if logged in as admin-->] You can import content channels from the Manage Content [<--with link] page.

[otherwise-->] You'll need to log in as an administrator to manage your content channels.

----

File: kolibri/plugins/learn/assets/src/actions.js [hunk 1]
```diff
@@ -115,17 +129,21 @@ function redirectToLearnChannel(store) {
...
+        store.dispatch('SET_CURRENT_CHANNEL', currentChannel);
+        cookiejs.set('currentChannel', currentChannel);
+        router.go(
+          {
+            name: constants.PageNames.LEARN_CHANNEL,
+            params: {
+              channel_id: currentChannel,
+            },
+          }
+        );
```
Comment (indirectlylit) [inline kolibri/plugins/learn/assets/src/actions.js]: these router calls can be tightened up a bit, e.g:

``` javascript
        router.go({
          name: constants.PageNames.LEARN_CHANNEL,
          params: { channel_id: currentChannel },
        });
```

**Comment by christianmemije** [comment]
@indirectlylit For now directly linked using /management/ since can't access the management constants from the learn plugin.

**Comment by indirectlylit** [comment]
👍

**Pull request merged**